docs/root/operations/admin.rst

```diff
@@ -770,6 +770,58 @@ modify different aspects of the server:
     Text readout stats create a new label value every time the value
     of the text readout stat changes, which could create an unbounded number of time series.
 
+  .. http:get:: /stats?format=prometheus&histogram_buckets=summary
+
+  Optional ``histogram_buckets`` query parameter is used to control how histogram metrics get reported.
+  If unset, histograms get reported as the "histogram" prometheus metric type, but can also be used to
+  emit prometheus "summary" metrics if set to ``summary``.
```

Comment:
I think it's worth documenting that each emitted summary is over the interval of the last stats flush interval (and link to `stats_flush_interval` docs).